jobs/kubectl/Dockerfile

```diff
@@ -1,6 +1,6 @@
 ARG BASE_IMAGE
 
-FROM bitnami/kubectl:1.31.1 as kubectl
+FROM bitnami/kubectl:1.33.3 as kubectl
```

Comment:
is there another one in `jobs/kubectl/Dockerfile.distroless` that needs to be updated?

Reply:
yes, good catch